tests/unit_tests/arelle/test_system_info.py

```diff
@@ -0,0 +1,7 @@
+"""Tests for system info."""
+from arelle.SystemInfo import get_system_info
+
+def test_get_system_info() -> None:
+    """Test that function get_system_info returns."""
+    get_system_info()
```

Comment:
Suggest verifying the expected keys are present

Reply:
Added in https://github.com/Arelle/Arelle/pull/360/commits/f63f58a818f177df61f8939e33c36f76ddf36ffd